test/unittests/end_to_end_test.cpp

```diff
@@ -31,7 +32,7 @@ TEST(end_to_end, milestone1)
     const auto wasm = from_hex(
         "0061736d0100000001070160027f7f017f030201000a13011101017f200020016a20026a220220006a0b");
     const auto module = parse(wasm);
-    auto instance = instantiate(module);
+    auto instance = instantiate(*module);
```

Comment:
I think the better change is to "inline" the `module` variable (CLion has a refactoring for that).
```suggestion
    auto instance = instantiate(parse(bin));
```

Reply:
Changed it everywhere, where possible, I think (i.e. where `module` variable is used only once in test)